extensions/yaml_support/labgraph_monitor/server/serializer_node.py

```diff
@@ -34,4 +45,4 @@ async def source(self) -> lg.AsyncPublisher:
                 stream_id=self.config.stream_id,
             )
             yield self.TOPIC, msg
-            await asyncio.sleep(1 / self.config.sample_rate)
+            await asyncio.sleep(1 / self.config.sample_rate), self.update(self.config.data, message.data)
```

Comment:
I was hoping that _awaiting_ the function `self.update()` will let me update the `self.config.data` before it actually creates `WSStreamMessage` at line 42, but it doesn't work. It still creates the `WSStreamMessage` without the updated `self.config.data`